### ISSUE TITLE
Last test missing from the report

### ISSUE THREAD

**Issue by sepa-joe** [Evilweed/protractor-beautiful-reporter#23]
Bit of a strange one here.

I have a set of some 90+ test specs and when I run them, the command line reports

**92 specs, 16 failures**

When I open the report, it shows only **15** fails (the exact last test is the one missing.)

To check this wasn't something really stupid I reran the exact same tests against the exact same codebase but this time used the jasmine2-html-reporter.  This showed the 16 test (and checked that the missing one was indeed present.)

Is this a known issue?

my config is below (just in case I am calling specs incorrectly - note this has the beautiful reporter commented out as this was during my test)

```
//var HtmlReporter = require('protractor-beautiful-reporter');
 var Jasmine2HtmlReporter = require('protractor-jasmine2-html-reporter');

exports.config = {
    framework: 'jasmine',
    
    // Capabilities to be passed to the webdriver instance.
    capabilities: {
        'browserName': 'chrome'
    },

	seleniumAddress: 'http://localhost:4444/wd/hub',
	
    specs: ['LA1-326\\guidance_page.js',
            'LA1-310\\question_1.js',        
            'LA1-310\\question_2.js',
            'LA1-310\\question_3.js',              
            'LA1-310\\question_4.js',              
            'LA1-310\\question_5.js',              
            'LA1-310\\question_6.js',
            'LA1-310\\question_7.js',
            'LA1-310\\question_8.js',
            'LA1-310\\question_9.js',
            'LA1-310\\question_10.js',
            'LA1-521\\first_name_validation.js',
            'LA1-521\\last_name_validation.js',
            'LA1-521\\email_validation.js',
            'LA1-521\\telephone_validation.js'],
    //specs: ['LA1-310\\question_7.js'],   

	baseUrl: 'http://localhost:4200',

	onPrepare: function() {
		jasmine.getEnv().addReporter(new Jasmine2HtmlReporter({
			savePath: 'reports/'
            })        
        );
    }

    /*onPrepare: function() {

        jasmine.getEnv().addReporter(new HtmlReporter({
            preserveDirectory: false,
            baseDirectory: 'reports2/',
            docTitle: 'SG Alpha 2',
            docName: 'index.html'
        }).getJasmine2Reporter());
    }*/
	
};
```

**Comment by ryan-gustafson** [comment]
I also noticed this issue, even with no failures.  The last spec in the last file was missing from the report.  All specs from all files are present with `protractor-jasmine2-html-reporter`.

**Comment by sepa-joe** [comment]
Yeah I have since seen that this also happens with no failures.  I'm actually glad someone else saw this - was beginning to think it was me :)

**Comment by schwarcu** [comment]
I also have this issue. Last executed "it" is always missing in HTML report. Other reporters don't have this issue. However they are ugly and unpractical. 

I would love to help but I don't really understand the code of this extension so I can't propose any solusion :<  Probably could be done by some dirty hack. For now I'm forced to add empty "it" step at the end of my tests.

**Comment by qanl** [comment]
I encountered the same issue.  Last test is missing from report altogether. Any plans to fix this?

**Comment by VenkatSaripilli** [comment]
Have the same issue. Adding an empty _it_ as the last test, as a temporary solutions to my tests.

**Comment by Evilweed** [comment]
@sepa-joe @ryan-gustafson @schwarcu @qanl @VenkatSaripilli Fixed in [v0.5.7](https://github.com/Evilweed/protractor-beautiful-reporter/releases/tag/v0.5.7)

@ryan-gustafson Thank You so much for using Your time to debug this!
